content/docs/next/upgrades/upgrade_3_3.md

```diff
@@ -399,7 +399,7 @@ _, err := kvc.Get(ctx, "a")
 +  if s.Code() == codes.Canceled
 ```
 
-[The new client balancer](https://github.com/etcd-io/etcd/blob/master/Documentation/learning/design-client.md) uses an asynchronous resolver to pass endpoints to the gRPC dial function. As a result, [v3.3.14](https://github.com/etcd-io/etcd/releases/tag/v3.3.14) or later requires `grpc.WithBlock` dial option to wait until the underlying connection is up.
+[The new client balancer](/docs/next/learning/design-client/) uses an asynchronous resolver to pass endpoints to the gRPC dial function. As a result, [v3.3.14](https://github.com/etcd-io/etcd/releases/tag/v3.3.14) or later requires `grpc.WithBlock` dial option to wait until the underlying connection is up.
```

Comment:
As I mentioned elsewhere, I'm very glad that you are fixing these external-meant-to-be-internal links. But in this case it exposes an issue with the "next" subsection: when we create a real version subsection from it (by copy-pasting the `next` subdirectory), links like these will need to be updated, right?

Reply:
I agree, links like this will need to be updated during the release process. We may be able to replace this with another front-matter parameter solution

Reply:
See #252